lib/symbols.js

```diff
@@ -59,7 +59,7 @@ const keys = {
   kChildLoggerFactory: Symbol('fastify.childLoggerFactory'),
   kHasBeenDecorated: Symbol('fastify.hasBeenDecorated'),
   kKeepAliveConnections: Symbol('fastify.keepAliveConnections'),
-  kRouteByFastify: Symbol('fastify.routeByFastify')
+  kRouteByFastify: Symbol('fastify.routeByFastify'),
```

Comment:
I'm noticing a lot of eslint inconsistencies - will open an issue

```suggestion
  kRouteByFastify: Symbol('fastify.routeByFastify')
```